Add error 1163 (#54)
mattpocock/ts-error-translator: packages/engine/errors/1163.md

---
original: "A 'yield' expression is only allowed in a generator body."

> excerpt: "Yield expressions must be used within a generator function"

Reviewer: ```suggestion
excerpt: "The `yield` keyword can only be used inside a generator function"
```

---

> If you want to use the functionality provided by yield, you should turn the enclosing function into a generator by adding an asterisk (*) next to the function name. Otherwise, you can remove the yield statement altogether.

Reviewer: Could you add an example usage?

Also, `the functionality provided by yield`, `enclosing function`, `yield statement` can be turned into friendlier language.
